src/encoded/tests/test_batch_download.py

```diff
@@ -423,34 +386,42 @@ def test_batch_download_lookup_column_value(lookup_column_value_item, lookup_col
ost/files/ENCFF010EPI/@@download/ENCFF010EPI.bam\nhttp://localhost/files/ENCFF790SUA/@@download/ENCFF790SUA.fastq.gz\nhttp://localhost/files/ENCFF002CON/@@download/ENCFF002CON.bam\nhttp://localhost/files/ENCFF009EPI/@@download/ENCFF009EPI.bam\nhttp://localhost/files/ENCFF001EPI/@@download/ENCFF001EPI.fastq.gz\nhttp://localhost/files/ENCFF002EPI/@@download/ENCFF002EPI.bam\nhttp://localhost/files/ENCFF558BPA/@@download/ENCFF558BPA.fastq.gz\nhttp://localhost/files/ENCFF011EPI/@@download/ENCFF011EPI.bam\nhttp://localhost/files/ENCFF001MRN/@@download/ENCFF001MRN.fastq.gz\nhttp://localhost/files/ENCFF002MRN/@@download/ENCFF002MRN.fastq.gz\nhttp://localhost/files/ENCFF003MRN/@@download/ENCFF003MRN.bam\nhttp://localhost/files/ENCFF004MRN/@@download/ENCFF004MRN.bam\nhttp://localhost/files/ENCFF005MRN/@@download/ENCFF005MRN.tsv\nhttp://localhost/files/ENCFF006MRN/@@download/ENCFF006MRN.tsv\nhttp://localhost/files/ENCFF807TRA/@@download/ENCFF807TRA.gtf.gz\nhttp://localhost/files/ENCFF107ADP/@@download/ENCFF107ADP.txt\nhttp://localhost/files/ENCFF003EPI/@@download/ENCFF003EPI.fastq.gz\nhttp://localhost/files/ENCFF004EPI/@@download/ENCFF004EPI.bam\nhttp://localhost/files/ENCFF001ISO/@@download/ENCFF001ISO.fastq.gz\nhttp://localhost/files/ENCFF002ISO/@@download/ENCFF002ISO.fastq.gz\nhttp://localhost/files/ENCFF003ISO/@@download/ENCFF003ISO.bam\nhttp://localhost/files/ENCFF004ISO/@@download/ENCFF004ISO.bam\nhttp://localhost/files/ENCFF005ISO/@@download/ENCFF005ISO.tsv\nhttp://localhost/files/ENCFF006ISO/@@download/ENCFF006ISO.tsv\nhttp://localhost/files/ENCFF007ISO/@@download/ENCFF007ISO.db\nhttp://localhost/files/ENCFF002BAR/@@download/ENCFF002BAR.tsv\nhttp://localhost/files/ENCFF005EPI/@@download/ENCFF005EPI.fastq.gz\nhttp://localhost/files/ENCFF002COS/@@download/ENCFF002COS.bed.gz\nhttp://localhost/files/ENCFF003COS/@@download/ENCFF003COS.bigBed\nhttp://localhost/files/ENCFF006EPI/@@download/ENCFF006EPI.fastq.gz\nhttp://localhost/files/ENCFF164JWT/@@download/ENCFF164JWT.fastq.gz\nhttp://localhost/files/ENCFF032OJW/@@download/ENCFF032OJW.bam\nhttp://localhost/files/ENCFF182CHA/@@download/ENCFF182CHA.bed.gz\nhttp://localhost/files/ENCFF007EPI/@@download/ENCFF007EPI.fastq.gz\nhttp://localhost/files/ENCFF001CON/@@download/ENCFF001CON.bam\nhttp://localhost/files/ENCFF003CON/@@download/ENCFF003CON.bam\nhttp://localhost/files/ENCFF001REP/@@download/ENCFF001REP.bam\nhttp://localhost/files/ENCFF003REP/@@download/ENCFF003REP.bam\nhttp://localhost/files/ENCFF008EPI/@@download/ENCFF008EPI.bam\nhttp://localhost/files/ENCFF959VGP/@@download/ENCFF959VGP.fastq.gz\nhttp://localhost/files/ENCFF477MLC/@@download/ENCFF477MLC.fastq.gz\nhttp://localhost/files/ENCFF000LBC/@@download/ENCFF000LBC.csqual.gz\nhttp://localhost/files/ENCFF000LBB/@@download/ENCFF000LBB.csqual.gz\nhttp://localhost/files/ENCFF000LBA/@@download/ENCFF000LBA.csfasta.gz\nhttp://localhost/files/ENCFF000LAZ/@@download/ENCFF000LAZ.csfasta.gz\nhttp://localhost/files/ENCFF010LAO/@@download/ENCFF010LAO.csfasta.gz'),
```

Comment:
I guess you think these tests are useless?

Reply:
They are sort of useful but they aren't quoted correctly (should have probably been caught in review). They are also a mess to deal with. Changing one letter invalidated the whole string.